components/button/floating-buttons.js

```diff
@@ -0,0 +1,236 @@
+import 'lit-media-query/lit-media-query.js';
```

Comment:
this replaces iron-media-query

Reply:
I think I need to install it somehow?

`Could not resolve module specifier "lit-media-query/lit-media-query.js" in file "/home/travis/build/BrightspaceUI/core/components/button/floating-buttons.js".

chrome 75                ✖ Test Suite Initialization

  Failed to resolve module specifier "lit-media-query/lit-media-query.js". Relative references must start with either "/", "./", or "../".`

Reply:
Yup you need it added to `package.json`, so in your git bash run `npm install --save lit-media-query` and that should install and add it.